plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -815,11 +804,132 @@ paths:
+                  code_hash:
+                    type: string
+                    description: A string representing the hash value of the specified account's smart contract code.
+
+  /get_transaction_id:
+    post:
+      description: Retrieves the transaction ID (also known as the transaction hash) of a specified transaction on the blockchain.
+      operationId: get_transaction_id
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              description: The transaction in JSON format for which the ID should be retrieved.
+              $ref: "https://docs.eosnetwork.com/openapi/v2.0/Transaction.yaml"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+                description: The transaction ID.
 
+  /get_producer_schedule:
+    post:
+      description: Retrieves the current producer schedule from the blockchain, which includes the list of active producers and their respective rotation schedule.
+      operationId: get_producer_schedule
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  active:
+                    description: A json object that encapsulates the list of active producers schedule and its version.
+                    $ref: "https://docs.eosnetwork.com/openapi/v2.0/ProducerSchedule.yaml"                              
+                  pending:
+                    description: A json object that encapsulates the list of pending producers schedule and its version.
+                    $ref: "https://docs.eosnetwork.com/openapi/v2.0/ProducerSchedule.yaml"
+                  proposed:
+                    description: A json object that encapsulates the list of proposed producers schedule and its version.
+                    $ref: "https://docs.eosnetwork.com/openapi/v2.0/ProducerSchedule.yaml"
+
+  /send_read_only_transaction:
+    post:
+      description: Sends a read-only transaction in JSON format to the blockchain. This transaction is not intended for inclusion in the blockchain. When a user sends a transaction with this option enabled, the connected node will execute the transaction and revert any state changes.
+      operationId: send_read_only_transaction
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                transaction:
+                  type: object
+                  properties:
+                    signatures:
+                      type: array
+                      description: array of signatures required to authorize transaction.
+                      items:
+                        $ref: "https://docs.eosnetwork.com/openapi/v2.0/Signature.yaml"
```

Comment:
The transaction is not allowed to have signatures. Or at least nothing is done with them.

Reply:
I took it out.